linera-witty-macros/src/lib.rs

```diff
@@ -7,6 +7,7 @@
 
```

Comment:
Any plan to add a unit test for the new derived code?

Reply:
I thought I did? Or maybe I didn't understand what you mean with new derived code?

Reply:
I meant quoting the derived code in a test inside `linera-witty-macros` (see `linera-views-macros`). This is not blocking, though, because we are indeed executing the derived code.